app/presenters/question_presenter.rb

```diff
@@ -1,29 +1,27 @@
-class QuestionPresenter
-  def initialize(question)
-    @question = question
-  end
+module QuestionPresenter
+  class << self
```

Comment:
Ok to use this for a module, since we assume most Module methods are Class methods.

For class, I highly recommend the `def self.method` syntax. This has the benefit to remind us which kind of method we are looking at, even when we're in the middle of a long file, without having to scroll back to the top. :)